### ISSUE TITLE
ALCLIFE markup incorrect

### ISSUE THREAD

**Issue by cusackjm** [episphere/questionnaire#222]
The ALCLIFE markup does not match the directions. Qx Document directions are: DISPLAY ALCLIFE IF ALCOHOL2=3
![image](https://user-images.githubusercontent.com/85582895/225678737-34c512fc-b67c-4ce6-b97e-207071049a95.png)

BUT, in the markup its: DISPLAY ALCLIFE IF ALCOHOL2=3 or =2. 
(participants that report 50-99 lifetime beverages are seeing lifetime questions, when they should not see this section)
![image](https://user-images.githubusercontent.com/85582895/225678361-0c88cae9-023d-4844-939f-f471608e3d11.png)

@FrogGirl1123 Do you want this to be on hold (for next prod push) or for this to be fixed now? @boyd-mj please wait for Nicole's response before starting this edit

**Comment by cusackjm** [comment]
based on Metrics as of 3/16/23 (100% response), we are going to update the questionnaire documents and keep the programming as is
